### DISCUSSION TITLE
Sony's ARW RAW able to covert JPEG preview but Hasselblad 3FR & HEIC RAW can't

### DISCUSSION THREAD

**Discussion by RayFungHK** [photoprism/photoprism#4515] [General]
![image](https://github.com/user-attachments/assets/97e86a7c-874e-402d-9d8b-d65cd1b4a4b8)
No preview show and stop scanning after 6 files, still no idea how to fix it...

**Comment by lastzero** [comment]
Information on which formats are supported can be found here by following the links:
- https://docs.photoprism.app/getting-started/faq/#what-media-file-types-are-supported
- https://web.photoprism.app/kb/file-formats

You can also try our preview build for improved HEIC support:
- https://github.com/photoprism/photoprism/issues/4439

**Reply by RayFungHK** [reply]
Updated, it can convert some 3FR preview thumbnail, but some may corrupt, and some may not be able to convert. Also it will stop converting, only 27 files converted although retry several scan times.
<img width="1196" alt="image" src="https://github.com/user-attachments/assets/0b6a0bdb-af5a-4faa-9854-e71033b73187">

**Reply by lastzero** [reply]
Since we use Darktable and/or RawTherapee for converting RAW files to JPEG, please check their documentation and issues for the status of 3FR file support:
- https://docs.darktable.org/usermanual/4.6/en/overview/supported-file-formats/
- https://www.darktable.org/resources/camera-support/
- https://www.libraw.org/supported-cameras

**Reply by lastzero** [reply]
Also make sure that you have enough memory and swap so that the conversion of RAW files does not fail because the process gets killed:
- https://docs.photoprism.app/getting-started/#system-requirements
- https://docs.photoprism.app/getting-started/troubleshooting/performance/#memory
